src/ParallelAlgorithms/ApparentHorizonFinder/Callbacks/IgnoreFailedApparentHorizon.hpp

```diff
@@ -29,7 +29,7 @@ struct IgnoreFailedApparentHorizon {
                     const FastFlow::Status failure_reason) {
     const auto& verbosity =
         db::get<logging::Tags::Verbosity<InterpolationTargetTag>>(box);
-    if (verbosity > ::Verbosity::Quiet) {
+    if (verbosity >= ::Verbosity::Quiet) {
```

Comment:
I know these clang-tidy errors aren't because of your change, but could you

```cpp
#include "IO/Logging/Tags.hpp"
#include "IO/Logging/Verbosity.hpp"
```

and hopefully they'll go away?